content/community/roadmap/studio/_index.en.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Roadmap for Altinn Studio
+linktitle: Studio
+description: High level roadmap and functional description for Altinn Studio and supporting functionality for the created apps.
+weight: 900
+aliases:
+- /roadmap-studio/
```

Comment:
We should use aliases only when moving pages inside docs.altinn.studio, never for new pages.